app/policies/linked_account_policy.rb

```diff
@@ -1,12 +1,12 @@
-class LinkedProfilePolicy < ApplicationPolicy
+class LinkedAccountPolicy < ApplicationPolicy
   alias_method :create?, :is_owner?
   alias_method :update?, :is_owner?
   alias_method :destroy?, :is_owner?
 
   class Scope < Scope
     def resolve
       return scope.where(private: false) unless user
```

Comment:
The entire scope method I think can be replaced with `scope.where(user: user)`, since you should only be able to see your own (and admins should not be allowed to peek)